packages/@aws-cdk/aws-route53-targets/lib/api-gateway-domain-name.ts

```diff
@@ -5,7 +5,7 @@ import * as route53 from '@aws-cdk/aws-route53';
  * Defines an API Gateway domain name as the alias target.
  *
  * Use the `ApiGateway` class if you wish to map the alias to an REST API with a
- * domain name defined throug the `RestApiProps.domainName` prop.
+ * domain name defined through the `RestApiProps.domainName` prop.
```

Comment:
appreciate the cleanup here (and elsewhere in this PR!).
in the future, let's save these for a different PR so they don't take focus away.

Reply:
Sorry, happy to remove it just noticed the typo when copying the existing target.

Reply:
it's fine to leave in, just a note for future contributions! 🙂